wrapper/src/main/java/software/amazon/jdbc/targetdriverdialect/PgTargetDriverDialect.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package software.amazon.jdbc.targetdriverdialect;
+
+import java.sql.Driver;
+import java.sql.SQLException;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Properties;
+import java.util.Set;
+import javax.sql.DataSource;
+import org.checkerframework.checker.nullness.qual.NonNull;
+import org.checkerframework.checker.nullness.qual.Nullable;
+import software.amazon.jdbc.HostSpec;
+
+public class PgTargetDriverDialect extends GenericTargetDriverDialect {
+
+  private static final Set<String> dataSourceClassMap = new HashSet<>(Arrays.asList(
+          "org.postgresql.ds.PGSimpleDataSource",
+          "org.postgresql.ds.PGPoolingDataSource",
+          "org.postgresql.ds.PGConnectionPoolDataSource"));
```

Comment:
Similar to Karens comment. Can we use `.getClass().getName()` in case package name or whatever changes?